changelog.md

```diff
@@ -12,6 +12,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ### Fixed
 - [Remove unused import from Http2StateUtil](https://github.com/ballerina-platform/ballerina-library/issues/5966)
+- [Document usage of Docker Compose](https://github.com/ballerina-platform/ballerina-library/issues/5972)
```

Comment:
The change log is maintained to list the changes done to the library package. Since this change is related to building the package, this is not required.